transaction/std/src/test_utils.rs

```diff
@@ -201,3 +262,28 @@ pub fn get_transaction<RNG: RngCore + CryptoRng, FPR: FogPubkeyResolver + Clone>
 
     transaction_builder.build(rng)
 }
+
+/// Build simulated change memo with zero amount
+pub fn build_zero_value_change_memo(
```

Comment:
I don't see this being used anywhere

Reply:
They're used in the memo builder tests to avoid a ton of boilerplate. I can see them being useful in future memo builder tests